hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapService.java

```diff
@@ -196,17 +160,14 @@ public void reset() {
 
     @Override
     public void shutdown(boolean terminate) {
-        if (config.isLiteMember()) {
-            return;
-        }
-
         for (PartitionContainer container : partitionContainers) {
             container.shutdown();
         }
+        antiEntropyFuture.cancel(true);
```

Comment:
Might need a null check for `antiEntropyFuture`.

Reply:
I added a test for `shutdown()` with and without `init()` (don't know if that race can happen). In that case we also need to have a null check on `container`. I've added both null checks, just to be sure.